src/components/header/Header.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Search from 'components/search';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import styles from './styles.module.css'
+
+const Header = () => {
+    return (
+        <header className={styles.header}>
+            <Container>
+                <Row>
+                    <Col>
+                        <a className={styles.logo} href="#">
```

Comment:
\<a\> element would be looking fine as a separate React component, reused in different places. Also you could utilize the principle of "Render props" inside this component, as there is a big part of elements to be passed as children in `src/components/item/MovieItem.jsx`:13